runtime/android/core_internal/src/org/xwalk/core/internal/XWalkUIClientInternal.java

```diff
@@ -94,6 +97,15 @@ public boolean onCreateWindowRequested(XWalkViewInternal view, InitiateByInterna
     }
 
     /**
+     * Called when the theme color is changed. This works only on Android Lollipop+(5.0+).
+     * @param color the new color in RGB format.
+     */
+    public void onDidChangeThemeColor(XWalkViewInternal view, int color) {
+        if (view == null || view.getActivity() == null) return;
+        ApiCompatibilityUtils.setStatusBarColor(view.getActivity().getWindow(),color);
```

Comment:
Question: As you said in log, this functionality only work 5.0+. Need we add API_version check here? Any exception/warning/error generated on 5.0 below Android?